nextcord/application_command.py

```diff
@@ -722,6 +761,12 @@ def from_callback(
         if self.name is None:
             self.name = self.callback.__name__
 
+        if self.error_callback is None:
+            if self.parent_cog:
+                self.error_callback = self.parent_cog._get_overridden_method(
+                    self.parent_cog.cog_application_command_error
+                )
+
```

Comment:
May I ask what purpose this serves here?

None of the values check / used appear to be passed into this method so isn't this just the same as calling this within `__init__`?

Reply:
It is required, because the init method doesn't get a parent_cog object with it, if the slash_command decorator is used.
But when the ClientCog object is initialized, it goes trough all the application commands it contains and updates them with its own object.

So this is required to make the decorator way of adding slash commands work.

Reply:
Sweet, thanks